bazel/ci/govulncheck.sh.in

```diff
@@ -24,18 +24,27 @@ submodules=$(${go} list -f '{{.Dir}}' -m)
 
 PATH=$(dirname "${go}"):${PATH}
 
-err=0
-
-echo "Scanning Go vulnerability DB for knwon vulnerabilities in modules:"
-for mod in ${submodules}; do
-  echo "  ${mod}"
-  echo -n "  "
-  CGO_ENABLED=0 ${govulncheck} "${mod}/..." |
-    tail -n 2 | # Providing some nice output...
-    tr '\n' ' ' |
-    sed s/" your code and"// &&
-    printf "\n" ||
-    err=$?
-done
-
-exit "${err}"
+check() {
+  err=0
+
+  echo "Scanning Go vulnerability DB for knwon vulnerabilities in modules:"
+  for mod in ${submodules}; do
+    echo "  ${mod}"
+    echo -n "  "
+    CGO_ENABLED=0 ${govulncheck} "${mod}/..." |
+      tail -n 2 | # Providing some nice output...
+      tr '\n' ' ' |
+      sed s/" your code and"// &&
+      printf "\n" ||
+      err=$?
+  done
+
+  exit "${err}"
+
+}
+
+if test -v silent; then
```

Comment:
```suggestion
if test -v SILENT; then
```

by convention, any environment variables that come from the outside (are not bash variables set locally in the script) should use all caps: https://google.github.io/styleguide/shellguide.html#constants-and-environment-variable-names